cmd/generate.go

```diff
@@ -234,7 +234,14 @@ func (gc *generateCmd) run() error {
 
 	customDataStr := templateGenerator.GetNodeBootstrappingPayload(gc.containerService, gc.containerService.Properties.AgentPoolProfiles[0])
 
-	cseCmdStr := templateGenerator.GetNodeBootstrappingCmd(gc.containerService, gc.containerService.Properties.AgentPoolProfiles[0], "<tenantid>", "<subid>", "rgname", "msiid")
+	cseCmdStr := templateGenerator.GetNodeBootstrappingCmd(gc.containerService, gc.containerService.Properties.AgentPoolProfiles[0],
```

Comment:
hmm how come I don't see this `GetNodeBootstrappingCmd` impl change in this PR?

Reply:
the change isn't in my PR, the build was broken tho. I don't think this is used anywhere anyway

Reply:
https://github.com/Azure/AgentBaker/commit/774d1d3b8755dd748faf016e2ec1a000721877ed#diff-a284f4316cf603f1f9b3cacaffc39a62R93